tools/StaticAnalysis/Exceptions/Az.EventHub/SignatureIssues.csv

```diff
@@ -34,3 +34,7 @@
 "Az.EventHub","New-AzEventHubThrottlingPolicyConfig","New-AzEventHubThrottlingPolicyConfig","1","8100","New-AzEventHubThrottlingPolicyConfig Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Az.EventHub","New-AzEventHubVirtualNetworkRuleConfig","New-AzEventHubVirtualNetworkRuleConfig","1","8100","New-AzEventHubVirtualNetworkRuleConfig Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Az.EventHub","Set-AzEventHubAuthorizationRule","Set-AzEventHubAuthorizationRule","1","8410","Parameter Rights of cmdlet Set-AzEventHubAuthorizationRule does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Az.EventHub","New-AzEventHubKeyVaultPropertiesObject","New-AzEventHubKeyVaultPropertiesObject","1","8100","New-AzEventHubKeyVaultPropertiesObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Az.EventHub","New-AzEventHubNamespaceV2","New-AzEventHubNamespaceV2","1","8410","Parameter MaximumThroughputUnits of cmdlet New-AzEventHubNamespaceV2 does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
```

Comment:
This error can not be suppressed for now unless it aligns with other code. Please change it to  MaximumThroughputUnit or ThroughputUnitsMaximum